### ISSUE TITLE
Research IIIF A/V capabilities for 3.0 Specs

### ISSUE THREAD

**Issue by DiegoPino** [esmero/format_strawberryfield#12]
# USE CASE

As our Media formatting options evolve and our use of IIIF becomes more and more transversal i want to start thinking of using IIIF to describe Audio and Video.

See https://iiif-commons.github.io/iiif-av-component/examples/data/iiif/mahler-symphony.json

# Discussion

Using the same concept/code we have for our Paged Formatter (IABook Reader) i would like to allow for the new Audio Formatter the use of a Metadata Display/Twig formatter to read a manifest and generated thought that the audio display, of course always allowing a fallback to read directly as:audio (code i already have)

I know Universal Viewer has Audio support and i guess Mirador 3 will eventually have A/V capabilities too

https://github.com/ProjectMirador/mirador/wiki

Regarding this, my main question would be:
A) a single manifest generated via Metadata Display Entities Twig that can serve Images/A/V and annotations + Search? That would mean basically integrate all media into Mirador for example for a unified experience, or 
B) one formatter per media type? As we have for books and now audio?
C) both options?
D) Always an HTML5 output option too, this is already coded and i like the simplicity a lot.
E) Search. This in particular would come out of the Formatter/Twig real and require a REST wrapper for our Solr Search, mostly like what @giancarlobi coded for Islandora IIIF module we have.

By the way, this my current metadata structure for audio

```JSON
{
"as:audio": {
"s3://f23/new-metadata-en-audio-58455d91acf7290275c1cab77531b7f561a11a84.mp3": 
  {
  "dr:fid": 32, 
  "dr:for": "some_audio_files", 
  "url": "s3://f23/new-metadata-en-image-58455d91acf7290275c1cab77531b7f561a11a84.mp3",
  "name": "My Super Audio",
  "type": "Audio",
  "encodingFormat": "audio/mpeg",
  "duration": "T0M15S", 
  "checksum": "f231aed5ae8c2e02ef0c5df6fe38a99b",
  "tracks": [
     {
     "subtitleLanguage": "es",
     "url": "s3://f11/subtitle-58455d91acf7290275c1cab77531b7f561a11a84.vtt", "dr:fid": 32, 
     "type": "subtitles|captions|descriptions|chapters|metadata",
     "dr:for": "some_track_files",
     }
   ]
  }}}
```

# Note

On dark nights like today i wish we had more community involvement! feels weird to make all these decisions without feedback, but then also, we can get their feedback/rollback or add more functionality once this is done, no harm really.

**Comment by DiegoPino** [comment]
@giancarlobi this is related:
https://github.com/cantaloupe-project/cantaloupe/issues/294

As we build new formatters, i started to like how we use IIIF / Cantaloupe to fetch all  image binaries so having a Passthrough processor started to make sense to me also for A/V. Maybe, as i say there, out of scope, but who knows :)

**Comment by giancarlobi** [comment]
@DiegoPino I think a Passthrough option is a really great idea!! In addition, this allows us to have, in the case, only one IIIF server to serve all DO requirements.

**Comment by DiegoPino** [comment]
@giancarlobi i think the scope creep that was mentioned (means making Cantaloupe more complex) is a valid concern. As i shared with you i think i will eventually write a IIIF Wrapper Proxy in Golang to go with this. Golang seems to be extremely powerful and easy to code for service/http based processing and there is also an S3 SDK available for that use case as backend for the files. This is in my wishlist but also out of scope of the current roadmap to release of a beta version. Will keep this Issue open. Thanks!

**Comment by DiegoPino** [comment]
Implemented since 1.0.0
